src/base/bittorrent/session.cpp

```diff
@@ -2160,6 +2134,27 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
         }
     }
 
+    // Common
```

Comment:
If possible, I would like this "common" block go before `if()` conditionals, i.e. above line 2106

Reply:
I keep it here because I want to bunch the similar if tests together.